### PR TITLE
Add support for File System Access API

### PR DESCRIPTION
**Pull request by vobruba-martin**
This API is enabled in Chrome 86 by default.

See https://www.chromestatus.com/feature/6284708426022912
See https://wicg.github.io/file-system-access/

### CODE REVIEW

**Comment by blickly** [comment]
Thanks for the PR!  I'll work on merging this in.

**Comment by vobruba-martin** [comment]
Hi @blickly , I found out that `AsyncIterable<string, !FileSystemHandle>` is wrong and the directory iterator should be typed [differently](https://wicg.github.io/file-system-access/#api-filesystemdirectoryhandle-asynciterable). I will update this PR tomorrow.

File: externs/browser/wicg_file_system_access.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+var FileSystemHandleKind;
+
+
+/**
+ * @interface
+ * @see https://wicg.github.io/file-system-access/#api-filesystemhandle
+ */
+var FileSystemHandle = function() {};
+
+/** @type {!FileSystemHandleKind} @const */
```
Comment (blickly) [inline externs/browser/wicg_file_system_access.js:34]: Nit, we often combine `@type {t} @const` into just `@const {t}`

File: externs/browser/wicg_file_system_access.js [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+FileSystemHandle.prototype.kind;
+
+/** @type {string} @const */
```
Comment (blickly) [inline externs/browser/wicg_file_system_access.js:37]: Same here

**Review by blickly — COMMENTED** [review]
OK, thanks for the heads up. I'll hold off for now, then.

Just a few minor style comments.

**Comment by vobruba-martin** [comment]
@blickly Done. If you need to squash commits let me know.

**Comment by blickly** [comment]
We've improved our process so that we no longer require squashing before merging.

I'll kick of the tests right now for starting the import.

**Comment by blickly** [comment]
Ok, looks good, thanks! I think I've found all the other folks in google who've written conflicting definitions and added updates for them.

The one last thing is I think we'll need to include a copyright in this file like the ones at the top of the other externs files, like:
https://github.com/google/closure-compiler/blob/master/externs/es3.js#L1-L15

**Comment by blickly** [comment]
OK, everything looks good. Looks like I missed getting this in today, but I think everything has been sorted out to submit on Monday.

**Comment by blickly** [comment]
Not sure why https://github.com/google/closure-compiler/commit/b8b0dd5bfde8428cf2a61d6724130cf0766e20b8 isn't listed as a merge of this PR, but I think it should be.

**Pull request closed without merging**